lib/simp/beaker_helpers.rb

```diff
@@ -86,8 +92,8 @@ def ensure_fixture_modules
 
 
   # Copy the local fixture modules (under `spec/fixtures/modules`) onto each SUT
-  def copy_fixture_modules_to( suts = hosts, opts = {})
-    ensure_fixture_modules
+  def copy_fixture_modules_to( suts = hosts, fixtures_yml = nil, opts = {})
```

Comment:
This API change has the potential to break *all* (or any) of our tests that are currently deployed. This needs to be analyzed across the entire code base prior to proceeding with this change.

It is also a breaking API change and would need to cause the version to bump to 2.X